docs/setup/postgresql.md

```diff
@@ -54,6 +54,11 @@ If the **Atlas** is deployed:
 psql -d georchestra -f postgresql/060-atlas.sql
 ```
 
+## GeoWebCache schema
+
+If geowebcache is used it creates a geowebcache schema to store quota and relevant infos to
+geowebcache. This setup is highly encouraged in a production environment to replace local H2 database.
```

Comment:
"... to replace the local H2 database"

Reply:
Missing psql statement with 110-geowebcache.sql  (for those who do the install manually)